### ISSUE TITLE
V4 Platform Ready not called on Cordova iOS/Android

### ISSUE THREAD

**Issue by orlyapps** [ionic-team/ionic#14489]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ ] **3.x**
[ x] **4.x**

**I'm submitting a ...**  (check one with "x")
[x ] bug report
[ ] feature request

**Current behavior:**
`this.platform.ready()`dont fire on cordova android/ios

**Expected behavior:**
Platform should fire a ready event

**Steps to reproduce:**
ionic start test (V4, blank, with cordova)
cd test
ionic cordova platform add android
ionic cordova run android

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/usr/local/lib/node_modules)

   @ionic/cli-utils  : 2.0.0-rc.6
   ionic (Ionic CLI) : 4.0.0-rc.6

global packages:

   cordova (Cordova CLI) : 8.0.0

local packages:

   @angular-devkit/core       : 0.6.0
   @angular-devkit/schematics : 0.6.0
   @angular/cli               : 6.0.1
   @ionic/schematics-angular  : 1.0.0-rc.6
   Cordova Platforms          : android 6.4.0
   Ionic Framework            : @ionic/angular 4.0.0-alpha.6

System:

   Android SDK Tools : 26.1.1
   ios-deploy        : 1.9.2
   NodeJS            : v8.11.2
   npm               : 6.0.1
   OS                : macOS High Sierra
   Xcode             : Xcode 9.3.1 Build version 9E501

Environment Variables:

   ANDROID_HOME : /Users/hackfrag/Library/Android/sdk
```

**Comment by orlyapps** [comment]
It also dont work with alpha 7 and android 7.0.0

https://github.com/ionic-team/ionic/blob/3bb661fa11e027f4cc55bd920d2353485beecff5/angular/src/providers/platform.ts#L49
Don't get called. Maybe its too late?

**Comment by paulstelzer** [comment]
At the moment platform.ready is not fire on Cordova, hope they fix it soon. At the moment you have to write your own code:

```
    document.addEventListener('deviceready', () => {
      /* Do what ever you want after device ready */
    });
```

**Comment by amreladawy** [comment]
May I ask, how do you inject platform as a dependency in your app component?

**Comment by paulstelzer** [comment]
They have fixed platform ready now -> #14577 

We only have to wait until the next alpha release ;) THis issue can closed now
